sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -174,3 +174,29 @@ def register_values(cls):
                  attrs.Source: [('SWPC', 'The Space Weather Prediction Center.')],
                  attrs.Provider: [('NOAA', 'The National Oceanic and Atmospheric Administration.')]}
         return adict
+    
+class WeatherDataClient(GenericClient):
```

Comment:
This needs a better name, like the other clients.

Reply:
> What does it mean by loading into TimeSeries, do you mean the folder ?

I don't understand why this reply is here but Timeseries is a class to read tabular data such as this.